.github/workflows/publish_release.yml

```diff
@@ -11,16 +11,16 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: Setup go env
         uses: actions/setup-go@master
         with:
-          go-version: "1.20"
+          go-version: "1.21"
       - name: goreleaser with tag
         uses: goreleaser/goreleaser-action@v5
         with:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Keep it as it is ! that new token is not necessary!

Reply:
Does the default GITHUB_TOKEN have  `write:packages` permission? If so we are fine

Reply:
`repo scope, write:packages`